newIDE/app/scripts/import-zipped-file.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * This script will extract a zipped file of a prebuilt external extension to GD that we don't want added to git history
+ * The zip file contains the raw, unchanged sources, which will be extracted into a folder
+ * The zip should be kept in a folder somewhere within the project
+ */
+var shell = require('shelljs');
```

Comment:
Similar as the previous comment, I think this file should be in newIDE/electron-app/scripts.